openedx/core/djangoapps/content/__init__.py

```diff
@@ -2,3 +2,7 @@
 Setup the signals on startup.
 """
 import openedx.core.djangoapps.content.course_structures.signals
+
+# Importing this in this manner seems to resolve some inexplicable circular import issue.
```

Comment:
This looks fragile.  "Inexplicable" circular import issues tend to be related to import order, and adding some import in some place happens to make the import order correct for the currently tested code.  However, the issue will come up again if someone refactors code somewhere, which then results in the code "inexplicably" breaking. :)
I'd prefer to track that down and fix it by using function-local imports instead; they are ugly, but better than this solution.